airflow/providers/google/cloud/triggers/kubernetes_engine.py

```diff
@@ -25,6 +25,7 @@
 from google.cloud.container_v1.types import Operation
 
 from airflow.exceptions import AirflowProviderDeprecationWarning
+from airflow.providers.cncf.kubernetes.callbacks import KubernetesPodOperatorCallback
```

Comment:
This will not work if user will have combination of newer google provider with older version of k8s provider.
I think we better wrap this with some mechanisem that detects this combo and warn the user that he needs to bump k8s provider to use this

Reply:
you're right; let me fix it.